<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="dataSourceStorageLocal">
    <data-source name="Django default" uuid="f3e563e5-1d2a-4889-8850-e604eb371931">
      <database-info product="" version="" jdbc-version="" driver-name="" driver-version="" />
      <introspection-schemas>*:gestionhc,kaladb</introspection-schemas>
    </data-source>
    <data-source name="kala" uuid="e8475431-1086-4016-bb6c-68932e23fd44">
      <database-info product="MySQL" version="5.5.45-log" jdbc-version="4.0" driver-name="MySQL Connector Java" driver-version="mysql-connector-java-5.1.40 ( Revision: 402933ef52cad9aa82624e80acbea46e3a701ce6 )">
        <extra-name-characters>#@</extra-name-characters>
        <identifier-quote-string>`</identifier-quote-string>
      </database-info>
      <case-sensitivity plain-identifiers="lower" quoted-identifiers="lower" />
      <user-name>b39a26ea8a2c83</user-name>
      <introspection-schemas>*</introspection-schemas>
    </data-source>
  </component>
</project>